backend/service/authn/authn.go

```diff
@@ -26,7 +26,13 @@ import (
 	"github.com/lyft/clutch/backend/service"
 )
 
-var scopes = []string{oidc.ScopeOpenID, "email"} // TODO(maybe): make scopes part of config?
+// TODO(maybe): make scopes part of config?
+// For more documentation on scopes see: https://developer.okta.com/docs/reference/api/oidc/#scopes
+var scopes = []string{
+	oidc.ScopeOpenID,
+	oidc.ScopeOfflineAccess, // offline_access is used to request issuance of a refresh_token
+	"email",
```

Comment:
Do we want to add in profile here as well so we request access to the end user's default profile claims like name?

Reply:
that someone could be me :)